src/scripts.js

```diff
@@ -32,9 +32,9 @@ let recipeData;
 let ingredientsData;
 
 let menuOpen = false;
-let recipes = [];
 let user;
 let pantry;
+let allRecipes = [];
```

Comment:
does this just take the place of the previous recipes array?

Reply:
yea, in my head this name made more sense than "recipes", since so many singular "recipe" variables are used later